### PR TITLE
Changed web description

### PR DESCRIPTION
**Pull request by robin-papp**
Changed description to "UniTandem provides higher education students in Finland with a chance to learn langauges and cultures through online tandem learning."

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="UniTandem provides higher education students in Finland with a chance to learn langauges and cultures through online tandem learning."
```
Comment (janakavishwajith) [inline public/index.html]: This text looks good to me.

**Review by janakavishwajith — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (EAkkanen) [reply public/index.html]: Looks good. There is a typo on "languages" tho!

**Review by EAkkanen — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (ayojava) [reply public/index.html]: It is okay

Comment (robin-papp) [reply public/index.html]: Thank you!

**Pull request merged**